apps/liquidity-provision-dashboard/src/app/components/dashboard/dashboard.tsx

```diff
@@ -0,0 +1,25 @@
+import { t } from '@vegaprotocol/react-helpers';
+
+import { Intro } from './intro';
+import { MarketList } from './market-list';
+
+export function Dashboard() {
+  return (
+    <>
+      <div className="px-16 pt-20 pb-12 bg-[#F0F0F0]">
```

Comment:
@mattrussell36 some new colours are used here, they are part of the styleguide Joelle has put together. Should I add them to a tailwind config or keep using them like this? If so, do I add them to the base config or only for this project?

Reply:
Definitely add to config and for now I'd say keep it in this projects config only